tests/e2e/http_health_check_test.go

```diff
@@ -158,6 +162,147 @@ func TestHTTPHealthHandler(t *testing.T) {
 	}
 }
 
+func TestHTTPLivezReadyzHandler(t *testing.T) {
+	e2e.BeforeTest(t)
+	client := &http.Client{}
+	tcs := []struct {
+		name           string
+		injectFailure  func(ctx context.Context, t *testing.T, clus *e2e.EtcdProcessCluster)
+		clusterOptions []e2e.EPClusterOption
+		healthChecks   []healthCheckConfig
+	}{
+		{
+			name:           "no failures", // happy case
```

Comment:
Should we have a check that confirms that both /readyz and livez check serializable_read?